.github/workflows/test.yml

```diff
@@ -38,8 +38,8 @@ jobs:
           pip install -e .[dev,cfg,z3]
       - name: Run tests
         run: |
-          pytest --cov python_ta --cov-config=.coveragerc --cov-report lcov --ignore=tests/test_type_constraints --ignore=tests/test_type_inference tests --ignore=tests/test_debug
-          pytest tests/test_debug
+          pytest tests --cov python_ta --cov-config=.coveragerc --cov-report lcov --ignore=tests/test_type_constraints --ignore=tests/test_type_inference --ignore=tests/test_debug
```

Comment:
This isn't the correct change. You don't need to move `tests`, but you _do_ need to modify the `--ignore=tests/test_debug` part. (Re-read my fourth bullet point carefully.)

Reply:
Also, add the `-vv` argument to this line (in addition to adding it to the next line, which is good).